### ISSUE TITLE
MiAIRR record on fairsharing.org

### ISSUE THREAD

**Issue by bcorrie** [airr-community/airr-standards#105]
Hi All,

Who is the owner of the MiAIRR record on fairsharing.org.

https://fairsharing.org/bsg-s000689

I noticed that the site is using the redirect to the SFU domain name rather than using the airr-community.org address. It should probably be changed from:

http://airr.irmacs.sfu.ca/working_groups/minimal_standards

to:

http://airr-community.org/working_groups/minimal_standards

Brian

**Comment by schristley** [comment]
Steve Kleinstein created the fairsharing.org record.

**Comment by ahmadchan** [comment]
Alejendra from oxford (http://www.oerc.ox.ac.uk/people/alejandra) is managing this portal and she suggested us (in one telecon) to update the standards page with documentation link and templates files  as well.

**Comment by bussec** [comment]
The fairsharing.org record currently refers to http://miairr.org, which redirects to this repository. The last decision on this topic from MiniStd WG (call on 2018-08-10) was to use http://miairr.airr-community.org, which also redirects here. The link http://airr-community.org/working_groups/minimal_standards does not work anymore, as it redirects to TAbS hence the path component breaks.
We should contact Steve once the current domain dispute is settled.

**Comment by bussec** [comment]
Contacted Steve today, asking whether he wants to continue updating the record or transfer this task to MiniStd. Will F/U on this here...

**Comment by schristley** [comment]
@bussec ping

**Comment by bussec** [comment]
Re-contacted Steve just now...

**Comment by bussec** [comment]
It turns out that the records cannot be "pushed" to other people, so I just sent a "claim" request for that record to fairsharing.org.

**Comment by bussec** [comment]
I now have access to the record, updated as many links as possible. Two questions before we can close this:
1. @bcorrie : The deep link you mentioned above (http://airr-community.org/working_groups/minimal_standards) is still not working. I cannot remember whether there was any final decision on this when we discussed the domain issues, but this is the URL we put into the NI paper...
2. @schristley @laserson : Is there a fairsharing record for the DataRep Standardized Representation? If yes, let me know and I will link it.

**Comment by schristley** [comment]
> Is there a fairsharing record for the DataRep Standardized Representation?

Not that I'm aware of.

**Comment by bcorrie** [comment]
> 1. @bcorrie : The deep link you mentioned above (http://airr-community.org/working_groups/minimal_standards) is still not working. I cannot remember whether there was any final decision on this when we discussed the domain issues, but this is the URL we put into the NI paper...

@bussec I believe you will need to talk to the Comms Committee about this, not sure where this has landed and if anything has been done since I was last involved in the URL redirect discussion - which was about this time last year...

**Comment by bussec** [comment]
@schristley @bcorrie I now included the DataRep (Front Immunol 2018) paper in the record. I thought about making the record a more general "AIRR standards" record, but realized that FAIRsharing makes a distinction between "Formats" and "Reporting Standards". So the record for the DataRep Standard is a new issue (#306). I close this ticket now, as the original question has been resolved.
